lib/identity/stateTransitions/identityCreateTransition/validateIdentityCreateSTDataFactory.js

```diff
@@ -32,7 +37,45 @@ function validateIdentityCreateSTDataFactory(dataProvider) {
       result.addError(new IdentityAlreadyExistsError(identityCreateTransition));
     }
 
-    // TODO: Here we need to fetch lock transaction, extract pubkey from it and verify signature
+    if (!result.isValid()) {
+      return result;
+    }
+
+    // fetch lock transaction, extract pubkey from it and verify signature
+
+    const lockedOutBuffer = Buffer.from(identityCreateTransition.getLockedOutPoint(), 'base64');
+    const { transactionHash, outputIndex } = Transaction.parseOutPointBuffer(lockedOutBuffer);
```

Comment:
We need to catch parsing errors and add consensus error if the outPoint Buffer format is wrong